docs/_data/en/toc_text.yml

```diff
@@ -10,6 +10,7 @@ shotgun: Shotgun
 python-api: Python API
 rest-api: REST API
 event-triggers: Event Driven Triggers
+jira-bridge: Shotgun Jira Bridge (Beta)
 examples: Examples
```

Comment:
Do we want to just call it "Jira Bridge" because it's already in the "Shotgun" section, so it's implied?

Reply:
sure!!! i'll fix it